openstates/ri/events.py

```diff
@@ -59,14 +59,20 @@ def scrape_agenda(self, url, session):
             "%A, %B %d, %Y %I:%M",
         ]
 
+        kwargs = {}
         if time in all_day:
             datetime = date
         else:
             datetime = "%s %s" % ( date, time )
-        if "CANCELLED" in datetime or "Rise of the House" in datetime:
-            # XXX: Do something more advanced.
+        if "CANCELLED" in datetime:
             return
 
+        if "Rise of the" in datetime:
+            datetime = date
+            kwargs["all_day"] = True
+            kwargs["time_text"] = time
```

Comment:
Yeah I guess that's a dumb idea. Description?

On Mon, Mar 9, 2015 at 2:09 PM, Rachel <notifications@github.com> wrote:

> In openstates/ri/events.py
> <https://github.com/sunlightlabs/openstates/pull/608#discussion_r26059915>
> :
>
> >              return
> >
> > +        if "Rise of the" in datetime:
> > +            datetime = date
> > +            kwargs["all_day"] = True
> > +            kwargs["time_text"] = time
>
> you mean the text (eg "Rise of the committee on whatever") in the event
> title?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunlightlabs/openstates/pull/608/files#r26059915>.
>



-- 
Paul Tagliamonte
Software Developer | Sunlight Foundation